posthog-node/test/feature-flags.spec.ts

```diff
@@ -2038,7 +2038,7 @@ describe('match properties', () => {
     ['is_date_after', '1h', '2022-04-30', false],
     // # Try all possible relative dates
     ['is_date_before', '1h', '2022-05-01 00:00:00', false],
```

Comment:
Can you add GMT here as well for consistency? I agree the right fix is to mock the timezone during tests and that it can be done some other time

Reply:
No problem! I wanted to fix all the dates, but blanketly adding `GMT` is not the correct fix. I'd have to do the date analysis on each of them. So I punted the rest of the dates for now. 😛